guides/source/active_storage_overview.md

```diff
@@ -961,10 +961,33 @@ class Uploader {
 
 NOTE: Using [Direct Uploads](#direct-uploads) can sometimes result in a file that uploads, but never attaches to a record. Consider [purging unattached uploads](#purging-unattached-uploads).
```

Comment:
Does this link need to be updated now that the headers have changed?

Reply:
No, but good pickup. That refers to a separate section that was added in https://github.com/rails/rails/commit/f4c51e8660bf1771268d45325fff8378eb90698e

![image](https://user-images.githubusercontent.com/509837/117359504-1e741080-ae7d-11eb-9b6e-78955444c885.png)